src/components/TopBar/CallButton.vue

```diff
@@ -253,6 +253,7 @@ export default {
 
 		startCallButtonDisabled() {
 			return this.disabled
+				|| this.$store.getters.callHasJustEnded
```

Comment:
I think it should be disabled to everyone except the actor (in case they closed by accident or there was an error)